packages/auth/src/tenant/model.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseModel } from '../shared/baseModel'
+
+export class Tenant extends BaseModel {
```

Comment:
Do we also need tenantSettings on auth ?

Reply:
I don't think so. The auth server just needs to be able to map identity providers to grants - I don't think anything else is becoming tenanted.